libraries/chain/transaction_context.cpp

```diff
@@ -339,13 +355,15 @@ namespace eosio { namespace chain {
          // NOTE: net_limit may possibly not be objective anymore due to net greylisting, but it should still be no greater than the truly objective net_limit
          net_limit = static_cast<uint64_t>(account_net_limit);
          net_limit_due_to_block = false;
+         use_resource_payer_net_limit = false;
```

Comment:
Is this right? I thought the idea was that resource payer account would be in play if specified.

Reply:
I think using resource payer limits when account (and other) limits are lower, will create a backdoor for abusing limits. But, I'm not sure what exactly the requirement is. I'll apply resource payer limits only, if that is what the requirement is. Need advice.